lib/chic/presents.rb

```diff
@@ -90,7 +90,7 @@ def _raise_presents_options_not_valid(message, attribute)
 
     def present(object, with: nil)
       if object.is_a?(Enumerable)
-        object.map { |o| present(o, with) }
+        object.map { |o| present(o, with:) }
```

Comment:
Value omission in hash literals isn't available in Ruby versions < 3.1 – so we'll need to pass in the value here explicitly because this gem requires Ruby >= 2.7.